utils/cache.py

```
@@ -146,13 +164,17 @@
             logger.debug(
                 f"Cache Hit for {self.primary_model.__name__} - Cache Key: {cache_key}"
             )
-            metrics.increment_cache(model=self.primary_model.__name__, type="hit")
+            metrics.increment_cache(
+                model=self.primary_model.__name__, cache_event_type="hit"
+            )
             return Response(cached_data, status=status.HTTP_200_OK)
         else:
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)